### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Budalebah**
current one not working , changed to ---> git clone https://github.com/NilFoundation/zkEVM-framework.git

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Framework for test verification of state of cluster based on EVM
 Using SSH:
 
 ```plain
-git clone git@github.com:NilFoundation/zkEVM-framework.git
+git clone https://github.com/NilFoundation/zkEVM-framework.git
```
Comment (akokoshn) [inline README.md:45]: `git clone git@github.com:NilFoundation/zkEVM-framework.git` is not link, it's command line for clone via github API. Works for me.

Comment (Budalebah) [reply README.md:45]: yes i know :) mean it wasnt work for me

Comment (akokoshn) [reply README.md:45]: it's required authentication. Your modification just looks like command on the 51th line

Comment (akokoshn) [reply README.md:45]: it's required authentication. Your modification just looks like command on the 51th line

**Comment by akokoshn** [comment]
@Budalebah , can we close this request?

**Comment by Budalebah** [comment]
Yes thank you

**Pull request closed without merging**